.github/workflows/release.yml

```
@@ -25,3 +27,68 @@
           Changelog:
           ${RELEASE_BODY}
           EOF
+      - id: build
+        env:
+          RELEASE_TAG_NAME: ${{ github.event.release.tag_name }}
+          DOCKER_REPO: ${{ vars.DOCKERHUB_REPO }}
+          DOCKER_USERNAME: ${{ vars.DOCKERHUB_USERNAME }}
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:3:32: Double quote to prevent globbing and word splitting

(shellcheck)